core/modules/wmgr/python/client.py

```diff
@@ -434,7 +471,9 @@ def loadData(self, dbName, tableName, dataFile, chunkId=None, overlap=False,
 
         @param dbName:     Database name
         @param tableName:  Table name
-        @param dataFile:   Path name for file with data
+        @param dataFile:   File object for file with data
+        @param fileName:   File name to use, if data is compressed then pass name which
+                           ends with .gz or set compressed argument to True.
```

Comment:
dataFile is a file object (the one returned from open() for example) representing actual data to be sent. fileName is a string with the file name, file name is not used to open the file, it is just sent to wmgr service so it can do something smart based on file name (e.g. uncompress it if it sees that name ends with .gz and you did not pass "sompressed" flag).